src/v2/Apps/_About2/AboutApp.tsx

```diff
@@ -18,235 +25,232 @@ export const AboutApp: React.FC = () => {
-        <Box mt={2} data-test="about1">
-          <Text variant="xl">THE ART WORLD ONLINE</Text>
+      <GridColumns mt={4}>
+        <Column span={6}>
+          <Text as="h1" variant="xl">
+            For the Love of Art
+          </Text>
+        </Column>
+        <Column span={6}>
           <Text variant="md">
-            Artsy features the world’s leading galleries, museum collections,
-            foundations, artist estates, art fairs, and benefit auctions, all in
-            one place. Our growing database of 500,000 images of art,
-            architecture, and design by 50,000 artists spans historical, modern,
-            and contemporary works, and includes the largest online database of
-            contemporary art. Artsy is used by art lovers, museum-goers,
-            patrons, collectors, students, and educators to discover, learn
-            about, and collect art.
+            Artsy envisions a future where everyone is moved by art every day.
+            To get there, we’re expanding the art market to support more artists
+            and art around the world. As the leading marketplace to discover,
+            buy, and sell fine art, Artsy believes that the process of buying
+            art should be as brilliant as art itself. That’s why we’re dedicated
+            to making a joyful, welcoming experience that connects collectors
+            with the artists and artworks they love.
           </Text>
-        </Box>
+        </Column>
+      </GridColumns>
 
-        <Box py={20} mt={2} data-test="about2">
-          <GridColumns>
-            <Column span={6}>
-              <Text variant="xs">ART ON ARTSY</Text>
-              <Text variant="lg">
-                We partner with leading galleries and institutions
-              </Text>
-              <Box py={2}>
-                <Image
-                  width="320px"
-                  height="471px"
-                  src="https://artsy-media-uploads.s3.amazonaws.com/tn07FUCvCfAPTdHLYksOcw/aboutPage_galleriePerotin.jpg"
-                />
-                <Text>Galerie Perrotin - Paris/Hong Kong/NY</Text>
-              </Box>
-              <Box py={2}>
-                <Image
-                  width="520px"
-                  height="299px"
-                  src="https://artsy-media-uploads.s3.amazonaws.com/fGoN2VtmO4BSETrSR6us5w/about_BritishMuseum.jpg"
-                />
-                <Text>The British Museum - London</Text>
-              </Box>
-              <Box py={2}>
-                <Image
-                  width="320px"
-                  height="480px"
-                  src="https://artsy-media-uploads.s3.amazonaws.com/8uU7UcN_OJ8Go3LUjwQ3FQ/about_cheimAndRead.jpg"
-                />
-                <Text>Cheim &amp; Read - NEW YORK</Text>
-              </Box>
+      <Rail
+        title="Discover Artworks Just for You"
+        subTitle="On Artsy"
+        getItems={() => {
+          const items: RailComponentProps[] = [
```

Comment:
One suggestion here: Rather than replicating the `ShelfArtwork` component by hand and including this data. You could just grab all the IDs for these artworks then write a query `{ artworks(ids: ["caleb-hahne-carry-me-home", ...etc]) { ...` which would let you use the real `ShelfArtwork` componentry. (Including things like the save button, correct metadata, etc.)

Reply:
@dzucconi is right here and this would def simplify things but will also require a bit of relay work. Happy to pair on this (or better yet we can do a follow-up PR implementing this suggestion)

Reply:
this isnt resolved yet but Damon and I are gonna pair on it in a future PR